### ISSUE TITLE
Unable to load a layout file as an anchor

### ISSUE THREAD

**Issue by maulik1shah** [google-ar/sceneform-android-sdk#832]
TLDR: I am in search of a working example that shows a layout file as an anchor, as the tutorial [example](https://developers.google.com/ar/develop/java/sceneform/create-renderables) does not work for me.

I have tried to follow the first part of tutorial on [Create a Renderable](https://developers.google.com/ar/develop/java/sceneform/create-renderables) page to render a text view as an anchor but was unable to load anything as an anchor. On the other hand, I am able to load a sphere as an anchor, but my end goal is to show anchors describing something about them. Do you have any working example of a layout file being shown as an anchor, as I am unable to find that in your samples?
